src/libZilliqa/Zilliqa.cpp

```diff
@@ -158,15 +158,17 @@ Zilliqa::Zilliqa(const PairOfKey& key, const Peer& peer, SyncType syncType,
   m_validator = make_shared<Validator>(m_mediator);
 
   if (LOOKUP_NODE_MODE) {
-    m_serverConnector = make_unique<SafeHttpServer>(RPC_PORT);
-
-  } else {
-    m_serverConnector = make_unique<SafeTcpSocketServer>(IP_TO_BIND, RPC_PORT);
+    m_lookupServerConnector = make_unique<SafeHttpServer>(LOOKUP_RPC_PORT);
+    m_lookupServer =
+        make_unique<LookupServer>(m_mediator, *m_lookupServerConnector);
   }
-  if (m_serverConnector == nullptr) {
-    LOG_GENERAL(FATAL, "m_serverConnector NULL");
+  m_statusServerConnector =
```

Comment:
Should we check for `ENABLE_STATUS_RPC`?

Reply:
Basically initialize m_statusServer to nullptr, and put lines 165 to 171 inside if (ENABLE_STATUS_RPC)

Reply:
can do